configs/edison/job_script.edison.bash

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash -l
+
+# comment out if using debug queue
+#SBATCH --partition=regular
+# comment in to get premium queue
+##SBATCH --qos=premium
+# comment in to get the debug queue
+##SBATCH --partition=debug
+# change number of nodes to change the number of parallel tasks
+#SBATCH --nodes=10
```

Comment:
so, let's see, in the current develop then generate=all would correspond to 10 tasks, right? Hence your choice of nodes=10 here. ok.
Yes, a comment explaining this and what you say about ntasks>nnodes would be great. thanks.